### PR TITLE
TIQR-287: Enforce multiple hosts, fix notification logo's

### PR DESCRIPTION
**Pull request by dzolnai**
Also allows proxies for acceptance builds

### CODE REVIEW

File: app/src/acceptance/res/xml/network_security_config.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config xmlns:tools="http://schemas.android.com/tools"
```
Comment (phavekes) [inline app/src/acceptance/res/xml/network_security_config.xml:2]: Why is this needed?

File: app/src/acceptance/res/xml/network_security_config.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    tools:ignore="InsecureBaseConfiguration">
+    <base-config cleartextTrafficPermitted="true">
```
Comment (phavekes) [inline app/src/acceptance/res/xml/network_security_config.xml:4]: This should not be needed in production

**Review by phavekes — CHANGES_REQUESTED** [review]

File: app/src/acceptance/res/xml/network_security_config.xml (hunk 2 above)
Comment (dzolnai) [reply app/src/acceptance/res/xml/network_security_config.xml:4]: It's not in production, it will only be used for acceptance build types (see the acceptance directory in its path)

File: app/src/acceptance/res/xml/network_security_config.xml (hunk 1 above)
Comment (dzolnai) [reply app/src/acceptance/res/xml/network_security_config.xml:2]: So we can inspect the HTTPS traffic using Charles or other proxy applications

File: app/src/acceptance/res/xml/network_security_config.xml (hunk 2 above)
Comment (phavekes) [reply app/src/acceptance/res/xml/network_security_config.xml:4]: Thank you for explaining

**Review by phavekes — APPROVED** [review]
